components/seventodos/actions/common.mjs

```diff
@@ -0,0 +1,43 @@
+import options from "../options.mjs";
```

Comment:
Please move these props into `seventodos.app.mjs` and make them `propDefinitions` so you can re use them inside `actions/create-task/create-task.mjs` as such. Then you can remove `options.mjs` file

Reply:
ok, thank you, just did your suggestion!